engine/gameobject/src/gameobject/comp_script.cpp

```diff
@@ -379,6 +379,48 @@ namespace dmGameObject
                 lua_setfield(L, action_table, "gamepad_name");
             }
 
+            if (params.m_InputAction->m_HasGamepadPacket)
```

Comment:
I'm tempted to put this in a action.gamepad table instead:

```
action.gamepad.axis
action.gamepad.hats
action.gamepad.buttons
```

Reply:
I think it is a good idea. Question is, what else should be put there?
Ofc we'll get some backwards compatibility issues if we move things without adding duplicate data. (e.g. device id)